(Doc+) Hotspotting link example troubleshooting video (#112932)
elastic/elasticsearch: docs/reference/troubleshooting/common-issues/hotspotting.asciidoc

ongoing significantly unique utilization may lead to cluster bottlenecks 
and should be reviewed. 

> See link:https://www.youtube.com/watch?v=Q5ODJ5nIKAM[this video for example troubleshooting walkthrough] of the below.

Reviewer: ```suggestion
See link:https://www.youtube.com/watch?v=Q5ODJ5nIKAM[this video] for a walkthrough of troubleshooting a hot spotting issue.
```
